native/nasm/Makefile

```diff
@@ -1,8 +1,8 @@
 PKG_NAME = nasm
-PKG_VERS = 2.14.02
+PKG_VERS = 2.15.05
 PKG_EXT = tar.gz
 PKG_DIST_NAME = $(PKG_NAME)-$(PKG_VERS).$(PKG_EXT)
-PKG_DIST_SITE = https://www.nasm.us/pub/nasm/releasebuilds/$(PKG_VERS)
+PKG_DIST_SITE = https://www.nasm.us/pub/$(PKG_NAME)/releasebuilds/$(PKG_VERS)
```

Comment:
_I am not a fan of using $(PKG_NAME) in url, as you cannot doubleclick the link. I know the links are invalid with $(PKG_VERS) anyway, but $(PKG_VERS) is variable, but $(PKG_NAME) is const within a package._ Just my own opignion.

Reply:
I am of the exact same opinion.  That specific change now reverted.